### PR TITLE
Add cobot_help.rst in Help

### PR DESCRIPTION
**Pull request by whosethink**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: Help/cobot_help.rst [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+cobot unassign <issus_link>
+
+::
+
+    cobot unssign https://github.com/coala/coala/issues/3845
+
+Create a Github issus given the title and repository
+-------------------------------------------------------
+
+cobot new issus
```
Comment (satwikkansal) [inline Help/cobot_help.rst]: typo, also mention about the alias `file` and the repository parameter for which the issue is filed,

 `cobot new issue coala-repository-name`

File: Help/cobot_help.rst [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+::
+
+    cobot new issus cobot "Add example of assigning an issue"
+
+Marks the given pull request as work in process/pending review
```
Comment (satwikkansal) [inline Help/cobot_help.rst]: "Marks" should be "Mark"

File: Help/cobot_help.rst [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+-----------------------------------------------------------------
+
+cobot mark (wip|pending) <pull URL>
+
+::
+
+    cobot mark wip https://github.com/coala/projects/pull/237
+
+Returns the info we have on your topic
+----------------------------------------
```
Comment (satwikkansal) [inline Help/cobot_help.rst]: Something like "Querying for explanation on some pre-defined topics" may be better.

**Comment by satwikkansal** [comment]
IMO this detailed documentation (if required) should be a part of cobot's repository https://gitlab.com/coala/cobot (as it'd be easier to maintain it there) and we should just reference it in the Newcomer docs.
Moreover as we already have `cobot help` I think documentation is redundant and will require extra efforts to maintain. So it's better to add a small description regarding how to use `cobot help` rather. I'd recommend you to discuss it on the issue or gitter channel :sweat_smile:

**Comment by Adrianzatreanu** [comment]
marking wip as per review given by @satwikkansal and the fact that it needs a rebase.

**Comment by jayvdb** [comment]
The idea behind this was rejected https://github.com/coala/coala/issues/3228

**Comment by jayvdb** [comment]
Duplicate of https://github.com/coala/documentation/pull/426

**Pull request closed without merging**